src/components/Footer.tsx

```diff
@@ -0,0 +1,75 @@
+import cn from 'classnames';
+import { Todo } from '../types/Todo';
+
+type Props = {
+  activeTodos: Todo[],
+  completedTodos: Todo[],
+  filteredTodos: Todo[],
+  onFilterAll: () => void,
+  onFilterActive: () => void,
+  onFilterCompleted: () => void,
+  selectedFilter: string,
+};
+
+export const Footer: React.FC<Props> = ({
+  activeTodos,
+  completedTodos,
+  onFilterAll,
+  onFilterActive,
+  onFilterCompleted,
+  selectedFilter,
+}) => {
+  return (
+    <footer className="todoapp__footer">
```

Comment:
as you don't do any calculations/fetches here you can return right away. But still I'd recommend to create some variables here and check whether they can improve readability of the component itself